ts/features/wallet/bancomat/component/bancomatCard/BaseBancomatCard.tsx

```diff
@@ -112,11 +125,24 @@ const BaseBancomatCard: React.FunctionComponent<Props> = (props: Props) => {
       {Platform.OS === "android" && <View style={styles.shadowBox} />}
       <View style={styles.cardBox}>
         <View>
-          {renderBankLogo(props.abi, imgDimensions)}
+          <View
+            style={{
+              flexDirection: "row",
+              alignItems: "center",
+              justifyContent: "space-between"
+            }}
+          >
+            {renderBankLogo(props.abi, imgDimensions)}
+            {props.blocked && (
+              <Badge style={[styles.badgeInfo, styles.badgeInfoExpired]}>
+                <H5 color="red">{I18n.t("messages.badge.expired")}</H5>
```

Comment:
blocked or expired?
I guess the badge should display "blocked" and not "expired"

Reply:
Right! it is blocked, i'm going to fix this